libsnark/gadgetlib1/gadgets/hashes/anemoi/anemoi_parameters.tcc

```diff
@@ -11,19 +11,745 @@
 
 namespace libsnark
 {
+using ppT = libff::bls12_381_pp;
```

Comment:
Yes, you are absolutely right! I was a bit too hasty in trying to "simplify" the syntax. Fixed now to explicitly state the types. As more curves are added I may move to your alternative suggestion to use a separate file per curve. Otherwise the file may become too large and difficult to manage.